core/commands/ipns.go

```diff
@@ -54,14 +54,6 @@ Resolve the value of another name:
 			return
 		}
 
-		if !n.OnlineMode() {
-			err := n.SetupOfflineRouting()
-			if err != nil {
-				res.SetError(err, cmds.ErrNormal)
-				return
-			}
-		}
-
```

Comment:
nope, don't do this. Loading public keys from disk for every command is slow. We only want to do it when necessary.